msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -0,0 +1,99 @@
+using System.IO;
+using System.Reflection;
+using Microsoft.Build.Utilities;
+using NUnit.Framework;
+using System.Globalization;
+using System.Threading;
+using System.Collections.Generic;
+
+namespace Xamarin.iOS.Tasks {
+	[TestFixture]
+	public class LocalizationStringTest : TestBase {
+
+		[Test]
+		public void OperatingSystemTranslations ()
+		{
+			CultureInfo culture1 = CultureInfo.CurrentCulture;
+			string locale = culture1.Name;
+
+			var task = CreateTask<CollectITunesArtwork> ();
+			task.ITunesArtwork = new TaskItem [] { new TaskItem (Assembly.GetExecutingAssembly ().Location) };
+
+			Assert.IsFalse (task.Execute (), "Execute failure");
+			Assert.AreEqual (1, Engine.Logger.ErrorEvents.Count, "ErrorCount");
+			bool isTranslated = TranslationAvailable (locale, Engine.Logger.ErrorEvents [0].Message);
+			Assert.IsTrue (isTranslated, "Your current locale is not supported: " + locale + ".");
```

Comment:
Why is this test supposed to fail with "Your current locale is not supported:"? The locale is never set for this test.

Reply:
In line 16, I am grabbing the User's current locale. I changed the name to be more appropriate and changed to the same getter I used in the second test "Thread.CurrentThread.CurrentUICulture". This test is just meant for someone to be able to see if their current locale they have set on their computer is compatible with our localization strings currently, but I can remove this test if that is not that useful!

Reply:
Ah, I see what you're doing. But I'm not sure it adds all that much value (if I finally understand it correctly, it will fail if we add more translated locales without updating the table of translations in this file, and then happen to run on a machine whose locale is one of the added translations - and that's seems both like a remote possibility and somewhat random behavior, so I think we can just remove this test).